src/types/genesys-cloud-media-session.ts

```diff
@@ -21,6 +21,7 @@ export class GenesysCloudMediaSession {
   private iceCandidatesReceivedFromPeer = 0;
   private interruptionStart?: Date;
   private logger: Logger;
+  private stateSyncTimeout?: any;
```

Comment:
Might be a NIT due to the severity of this but we were trying to move away from `any` right?

Reply:
I messed with it but I was getting compiler errors when running the tests. `NodeJS.Timers` can be annoying to deal with so I just made it an `any`.